app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt

```diff
@@ -357,9 +360,20 @@ class PatientsTabScreen : BaseScreen<
   }
 
   override fun showDrugStockReminderCard() {
+    val previousMonthAndYear = previousMonthAndYear()
+    drugStockReminderCardSubTitle.text = resources.getString(R.string.drug_stock_reminder_card_content, previousMonthAndYear)
+
     showHomeScreenBackground(R.id.drugStockReminderCardLayout)
   }
 
+  private fun previousMonthAndYear(): String {
+    val localDate = LocalDate.now(userClock).minusMonths(1)
+    val previousMonth = localDate.month.toString()
+    val capitalizedFirstLetterOfTheMonth = previousMonth.uppercase().subSequence(0, 1)
+    val lowercaseTheRestOfTheMonthName = previousMonth.lowercase().subSequence(1, 3)
+    return "${capitalizedFirstLetterOfTheMonth}${lowercaseTheRestOfTheMonthName}-${localDate.year}"
```

Comment:
One of the reasons why this approach can be problematic is because of localization. This approach always prints the month in English. Since we support multiple calendar formats, it's better to let the `DateTimeFormatter` handle it.

Reply:
Is datetimeformatter being localised?

Reply:
TIL